chainer/utils/error.py

```diff
@@ -0,0 +1,9 @@
+def _format_array_props(arrays):
+    # Formats array shapes and dtypes for error messages.
+    def format_props(arr):
+        return '{}:{}'.format(arr.shape, arr.dtype.name)
```

Comment:
This local function is not used so we can remove it.

Reply:
Fixed.